lib/fastlane/plugin/wpmreleasetoolkit/helper/generate_po_file_from_metadata_helper.rb

```diff
@@ -0,0 +1,80 @@
+require 'gettext/po'
+
+module Fastlane
+  module Helper
+    # Helper methods to execute PO related operations
+    #
+    module GeneratePoFileMetadataHelper
+
+      def self.add_header_to_po(po_obj)
+        po_obj[''] = <<~HEADER
```

Comment:
I've looked at a number of `po` files and there does not seem to be consistency about what the header should look like. Would this simple header work?

I have also noticed that for example [PlayStoreStrings for Jetpack did not update the date ](https://github.com/wordpress-mobile/WordPress-Android/blob/trunk/WordPress/jetpack_metadata/PlayStoreStrings.po#L5) so I have left `PO-Revision-Date:` out for now.